deeppavlov/configs/elmo/elmo_paraphraser_fine_tuning.json

```diff
@@ -0,0 +1,79 @@
+{
+  "dataset_reader": {
+    "class_name": "file_paths_reader",
+    "data_path": "{DOWNLOADS_PATH}/paraphraser_train_and_pretrain_texts/",
+    "train": "paraphraser_train_and_pretrain_texts_train.txt",
+    "valid": "paraphraser_train_and_pretrain_texts_valid.txt",
+    "test": "paraphraser_train_and_pretrain_texts_test.txt"
+  },
+  "dataset_iterator": {
+    "class_name": "elmo_file_paths_iterator",
+    "seed": 31415,
+    "unroll_steps": 20,
+    "max_word_length": 50,
+    "n_gpus": 1,
+    "shuffle": false,
+    "bos":"<S>",
+    "eos":"</S>",
+    "save_path": "{MODELS_PATH}/elmo-1b-benchmark/vocab-2016-09-10.txt",
+    "load_path":"{MODELS_PATH}/elmo-1b-benchmark/vocab-2016-09-10.txt"
+  },
+  "chainer": {
+    "in": [
+      "x_char_ids"
+    ],
+    "in_y": [
+      "y_token_ids"
+    ],
+    "pipe": [
+      {
+        "class_name": "elmo_model",
+        "options_json_path": "{MODELS_PATH}/elmo-1b-benchmark/options.json",
+        "unroll_steps": 20,
+        "batch_size": 128,
+        "save_path": "{MODELS_PATH}/elmo-1b-benchmark/saves/model",
+        "load_path": "{MODELS_PATH}/elmo-1b-benchmark/saves/model",
+        "in": ["x_char_ids", "y_token_ids"],
+        "in_y": [],
+        "n_gpus": 1,
+        "out": ["loss"]
+      }
+    ],
+   "out": [
+    "x_char_ids",
+    "y_token_ids"
+  ]
+  },
+  "train": {
+    "epochs": 2,
+    "batch_size": 128,
+    "log_every_n_batches": 24,
+    "val_every_n_epochs": 1,
+    "validation_patience": 1,
+    "metric_optimization": "minimize",
+    "metrics": [
+      {
+        "name": "elmo_loss2ppl",
+        "inputs": ["loss"]
+      }
+    ],
+    "tensorboard_log_dir": "{MODELS_PATH}/elmo-1b-benchmark/logs"
+  },
+  "metadata": {
+    "variables": {
+      "ROOT_PATH": "~/.deeppavlov",
+      "DOWNLOADS_PATH": "{ROOT_PATH}/downloads",
+      "MODELS_PATH": "{ROOT_PATH}/models"
+    },
+    "requirements": [
+      "{DEEPPAVLOV_PATH}/requirements/tf.txt",
+      "{DEEPPAVLOV_PATH}/requirements/tf-hub.txt"
+    ],
+    "download": [
+      {
+        "url": "http://files.deeppavlov.ai/deeppavlov_data/paraphraser_train_and_pretrain_texts.tar.gz",
+        "subdir": "{DOWNLOADS_PATH}/paraphraser_train_and_pretrain_texts"
+      }
```

Comment:
If this config is used for fine-tuning an elmo model, shouldn't it download the pretrained one first?

Reply:
added download.